app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java

```diff
@@ -349,6 +393,78 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+
+    public static class DatePickerFragment extends DialogFragment {
+
+        private final Calendar c;
```

Comment:
You could use better variable name like calendar.

Reply:
Ok thanks!

Reply:
Thank you.